Quantum-Control-Applications/Optically addressable spin qubits/NV center in a confocal setup/g2_MZI_experiment.py

```diff
@@ -0,0 +1,132 @@
+"""
+g2_MZI_experiment.py: Single NVs confocal microscopy - Intensity auto-correlation g2 using 2 channels in a Mach-Zender Interferometer.
+"""
+
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm.qua import *
+from qm import SimulationConfig
+from configuration import *
+import matplotlib.pyplot as plt
+
+
+# Add SPCM2 to the config if it is not already there
+from copy import deepcopy as _deepcopy
+
+config["controllers"]["con1"]["analog_inputs"][2] = {"offset": 0}  # Add analog input 2
+config["controllers"]["con1"]["digital_outputs"][3] = {}  # Add digital output 3
+config["elements"]["SPCM2"] = _deepcopy(config["elements"]["SPCM"])  # Copy SPCM element
+config["elements"]["SPCM2"]["outputs"] = {"out1": ("con1", 2)}  # Map SPCM2 to analog input 2
+config["elements"]["SPCM2"]["digitalInputs"]["marker"]["port"] = ("con1", 3)  # Map SPCM2 to digital output 3
```

Comment:
Why do we just not add it to the config?
This is what we do for most scripts.

Reply:
The disadvantage of this method:
If someone already has a config, this might collide with what he already has inside.